lib/grape/error_formatter/json.rb

```diff
@@ -3,14 +3,23 @@ module ErrorFormatter
     module Json
       class << self
         def call(message, backtrace, options = {}, env = nil)
-          message = Grape::ErrorFormatter::Base.present(message, env)
+          result = wrap_message(Grape::ErrorFormatter::Base.present(message, env))
 
-          result = message.is_a?(String) ? { error: message } : message
           if (options[:rescue_options] || {})[:backtrace] && backtrace && !backtrace.empty?
             result = result.merge(backtrace: backtrace)
           end
           MultiJson.dump(result)
         end
+
+        private
+
+        def wrap_message(message)
+          if message.is_a?(Exceptions::ValidationErrors)
+            return message
```

Comment:
Yeah you're right that conditional was sloppy.  Corrected.  Will squash if this looks better.